releng/setup-env.sh

```diff
@@ -6,6 +6,8 @@ build_platform=$(uname -s | tr '[A-Z]' '[a-z]' | sed 's,^darwin$,macos,')
 build_arch=$($releng_path/detect-arch.sh)
 build_platform_arch=${build_platform}-${build_arch}
 
+python_loc=$(which python3)
```

Comment:
- Perhaps we should name this just `python` or `python_interpreter`?
- Would be good to pass this in from the Makefiles, so the value of `PYTHON3` from the `config.mk` is used instead.